src/main/java/mate/academy/internetshop/controllers/CartProductRemoveController.java

```diff
@@ -0,0 +1,25 @@
+package mate.academy.internetshop.controllers;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.academy.internetshop.lib.Injector;
+import mate.academy.internetshop.service.ProductService;
+import mate.academy.internetshop.service.ShoppingCartService;
+
+public class CartProductRemoveController extends HttpServlet {
+    private static final Injector INJECTOR = Injector.getInstance("mate.academy.internetshop");
+    private final ShoppingCartService shoppingCartService =
+            (ShoppingCartService) INJECTOR.getInstance(ShoppingCartService.class);
+    private final ProductService productService
+            = (ProductService) INJECTOR.getInstance(ProductService.class);
+
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        String productId = req.getParameter("id");
+        Long id = Long.valueOf(productId);
```

Comment:
have finished this controller?

Reply:
I haven't, that's a template for the next homework, I will be changing it